x/bank/module.go

```diff
@@ -99,14 +104,17 @@ type AppModule struct {
 
 	keeper        keeper.Keeper
 	accountKeeper types.AccountKeeper
+
+	// legacySubspace is used solely for migration of x/params managed parameters
+	legacySubspace exported.Subspace
 }
 
 // RegisterServices registers module services.
 func (am AppModule) RegisterServices(cfg module.Configurator) {
 	types.RegisterMsgServer(cfg.MsgServer(), keeper.NewMsgServerImpl(am.keeper))
 	types.RegisterQueryServer(cfg.QueryServer(), am.keeper)
 
-	m := keeper.NewMigrator(am.keeper.(keeper.BaseKeeper))
+	m := keeper.NewMigrator(am.keeper.(keeper.BaseKeeper), am.legacySubspace)
```

Comment:
I think we're missing the migration call here :)

Reply:
I am actually trying to fit in within `Migrate3to4` (without much success yet) as we have ConsensusVersion `3` in v0.46, so I don't think we need to bump it for the SendEnabled PR and for the params migrations. Do we?

Reply:
ahh right, good point!